recipes/npcap/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from os.path import join
+from conan import ConanFile
+from conan.tools.files import get, download, copy
+
+
+class NpcapConan(ConanFile):
+    name = "npcap"
+    description = "Windows port of the libpcap library"
+    homepage = "https://npcap.com/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "https://github.com/nmap/npcap/blob/master/LICENSE"
+    topics = ("npcap", "pcap")
+    settings = {
+        "os": ["Windows"],
+        "arch": ["x86", "x86_64", "armv8"],
+        "build_type": None,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self._source_subfolder)
+        download(self, filename="LICENSE", url=[
+            f"https://raw.githubusercontent.com/nmap/npcap/v{self.version}/LICENSE",
```

Comment:
move it to conandata.yml. sha256 should be check. Use https://github.com/conan-io/conan-center-index/blob/master/recipes/approvaltests.cpp/all/conandata.yml as example.